Doc/library/dataclasses.rst

```diff
@@ -31,7 +31,7 @@ using :pep:`526` type annotations.  For example, this code::
       def total_cost(self) -> float:
           return self.unit_price * self.quantity_on_hand
 
-will add, among other things, a :meth:`~object.__init__` that looks like::
+will add, among other things, a :meth:`~!object.__init__` that looks like::
```

Comment:
while we're here:

```suggestion
will add, among other things, an :meth:`~!object.__init__` that looks like::
```

Reply:
This is a whole other can of worms:  is it "an init" or "a dunder init"?  This page waffles on it.

Reply:
as you wish :)

Reply:
Let's see what the public thinks: https://hachyderm.io/@nedbat/112129459668043424